docs/integrations/builtin/core-nodes/n8n-nodes-base.n8n.md

```diff
@@ -136,7 +136,7 @@ Configure this operation with this parameter:
 	* `connections`
 	* `settings`
 
-Refer to the [n8n API | Create a workflow documentation](/api/api-reference/#tag/Workflow/paths/~1workflows/post) for more information.
+Refer to the [n8n API | Create a workflow documentation](/api/api-reference.md/#tag/Workflow/paths/~1workflows/post) for more information.
```

Comment:
@evilnick did this actually work locally? I thought I'd tried this but wasn't able to get it to build the link correctly, but I might be misremembering.

Reply:
I thought I would give it a go and it did indeed work, but i want to see the preview

Reply:
@imchairmanm :partying_face:

Reply:
Ah, looks like it builds it but does note that it cannot find the anchor tag:

```
4:46:42 PM: INFO    -  Doc file 'integrations/builtin/core-nodes/n8n-nodes-base.n8n.md' contains a link '/api/api-reference.md/#tag/Workflow/paths/~1workflows/post', but the doc 'api/api-reference.md' does not contain an anchor '#tag/Workflow/paths/~1workflows/post'.
```

Better than I thought it was :+1: thanks for verifying.

Reply:
Oh yes, it still whines about the anchor. I updated one external redirect as well - I ran the deploy site through an external linkchecker - there are a dozen or so more we should follow up on sometime

Reply:
Oh, that's excellent. Glad we are getting some info on external links as well.